src/crimson/os/seastore/segment_cleaner.h

```diff
@@ -114,8 +114,12 @@ class segments_info_t {
     return segment_size;
   }
   std::size_t get_num_in_journal() const {
```

Comment:
This is actually not the number of segments that are "in journal" -- segments between journal-head and journal-tail. Instead, it is the number of segments of type `segment_type_t::JOURNAL`, will fix it.

Reply:
fixed by https://github.com/ceph/ceph/pull/46158/commits/bb2ad82466b5eb4dc3a860e35cf522312da54d18